### DISCUSSION TITLE
Unable to display timestamp format from ActiveSpaces

### DISCUSSION THREAD

**Discussion by taycwee** [dbeaver/dbeaver#17583] [Q&A]
I'm unable to change the timestamp format displayed in Dbeaver. 
![image](https://user-images.githubusercontent.com/58674780/188827102-93ed3427-9b57-4217-97b8-ceda3e82e66e.png)

Corresponding display using SQL workbench. 
![image](https://user-images.githubusercontent.com/58674780/188827361-edae1ddb-1892-48f4-b7b1-fec63d4a8ce6.png)

Is there any means to convert the following format to the standard format? 
`{sec=1626457021, nsec=0}`

**Comment by LonwoLonwo** [comment]
Hello @taycwee 

What database do you use?

Also, you can try to change your data format in Preferences -> Editors -> Data Editor -> Data Formats

**Reply by taycwee** [reply]
I'm using ActiveSpace IMDG

**Reply by taycwee** [reply]
![image](https://user-images.githubusercontent.com/58674780/189014447-4be1366f-fdca-4244-8ae2-ba1cf899a24f.png)

I have tried that. It seems like Dbeaver does not recognize the timestamp format from IMDG (ActiveSpaces).

**Reply by LonwoLonwo** [reply]
And what driver do you use? First, it is a driver's work to return data in the proper format.

**Reply by taycwee** [reply]
I see. I'm currently using the driver from Tibco datagrid (com.tibco.datagrid.jdbc.DriverImpl).
